.gitignore

```diff
@@ -10,4 +10,8 @@ MUJOCO_LOG.TXT
 # build
 build
 dist
-*.egg-info
\ No newline at end of file
+*.egg-info
+
+# data
+data/distillation_buffer*.pkl
```

Comment:
According to this comment https://github.com/medipixel/rl_algorithms/pull/259/files#r452582414,
should remove useless ignore.